mkosi.conf.d/30-debian-ubuntu/mkosi.conf

```diff
@@ -8,30 +8,26 @@ Distribution=|ubuntu
 Packages=
         ?exact-name(systemd-ukify)
         apt
-        archlinux-keyring
         bash
         btrfs-progs
         bubblewrap
         ca-certificates
         coreutils
         cpio
         curl
-        dbus-broker
+        dbus
```

Comment:
I think it would be better if we had `dbus` in a separate file just for focal. I don't think `dbus` will necessarily be installed in a dbus-broker-only system (though weird stuff will pull it in anyway via some longwinded chain), so it might be better to not ask for it in all cases.